synapse/http/server.py

```diff
@@ -271,7 +276,7 @@ async def _async_render_wrapper(self, request: SynapseRequest):
             f = failure.Failure()
             self._send_error_response(f, request)
 
-    async def _async_render(self, request: Request):
+    async def _async_render(self, request: SynapseRequest) -> Optional[Tuple[int, Any]]:  # type: ignore[return]
```

Comment:
What complaint is being suppressed here?

Reply:
Seems like it isn't needed. Not sure why I added it!